mkosi

```diff
@@ -79,6 +79,73 @@ MKOSI_COMMANDS_SUDO = ("build", "clean") + MKOSI_COMMANDS_CMDLINE
 MKOSI_COMMANDS = ("build", "clean", "help", "summary") + MKOSI_COMMANDS_CMDLINE
 
 
+DRACUT_UNIFIED_KERNEL_INSTALL = """\
+#!/bin/bash -e
+
+COMMAND="$1"
+KERNEL_VERSION="$2"
+BOOT_DIR_ABS="$3"
+KERNEL_IMAGE="$4"
+ROOTHASH="${5:-}"
```

Comment:
I have only checked the `kernel-install` man page and it only lists four arguments; will the fifth be passed through?

Reply:
The ROOTHASH is only there so we can reuse the script in `install_unified_kernel` where we invoke it directly with the extra roothash parameter. In general kernel-install operation, it won't get used although kernel-install also passes extra arguments to the scripts it invokes so in theory we could pass calculate the roothash in the pacman hook and pass it to kernel-install as well. However, roothash is for dm-verity in which case the rootfs is readonly so passing it via the hook isn't really useful. It's still useful for reusing the logic in `install_unified_kernel` though.